pkg/operations/kubernetesupgrade/upgradecluster.go

```diff
@@ -185,7 +195,7 @@ func (uc *UpgradeCluster) getUpgradeWorkflow(kubeConfig string, aksEngineVersion
 	return u
 }
 
-func (uc *UpgradeCluster) getClusterNodeStatus(kubeClient kubernetes.Client, resourceGroup string) error {
+func (uc *UpgradeCluster) getClusterVMStatus(kubeClient kubernetes.Client, resourceGroup string) error {
```

Comment:
I read through this method, and I think a clearer new name would be something like `setNodesToUpgrade`.